website/sidebarsIntegrations.js

```diff
@@ -156,16 +153,6 @@ module.exports = {
                         "sources/freeipa/index",
                     ],
                 },
-                "sources/general",
```

Comment:
Perhaps we could add a pointer to their new location in `sources/index`

Reply:
Good point, we should probably also add that for the integrations section above